Source/WebCore/rendering/LegacyInlineTextBox.h

```diff
@@ -165,10 +169,20 @@ class LegacyInlineTextBox : public LegacyInlineBox {
     // Where to truncate when text overflow is applied.
     std::optional<unsigned short> m_truncation;
 
+    bool m_isInGlyphDisplayListCache : 1 { false };
```

Comment:
This might mess up the struct layout (but maybe we don't care).

Reply:
I moved this to the other flags up in `LegacyInlineBox`.